openscap_report/scap_results_parser/parsers/rule_parser.py

```diff
@@ -1,13 +1,41 @@
 # Copyright 2022, Red Hat, Inc.
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
+import collections
 from dataclasses import replace
 
 from ..data_structures import Identifier, Reference, Rule, RuleWarning
 from ..namespaces import NAMESPACES
 from .full_text_parser import FullTextParser
 from .remediation_parser import RemediationParser
 
+KNOWN_REFERENCES = {
```

Comment:
Please add comments to the code to turn off long-line warnings for this constant.

Reply:
I have add the comments.

Reply:
Pylint is not happy.  Please use `# pylint: disable=line-too-long` before the constant definition and `# pylint: enable=line-too-long` after the constant definition. You can check this with the `tox -e code_style` command.